scripts/drtprod

```diff
@@ -331,6 +341,15 @@ EOF"
         # setup dns
         $0 dns drt-chaos create
         ;;
+      "workload-chaos")
+        roachprod create workload-large \
+          --clouds gce \
```

Comment:
We should probably still throw in a `--zone us-east1-c` so we at least end up in the same region as drt-chaos.

Reply:
Done